duckduckgo/iOS
# Feature/brad/js api overrides

Task/Issue URL:https://app.asana.com/0/0/1195460247972058/f
Tech Design URL: https://app.asana.com/0/481882893211075/1199166946567550/f
CC: @brindy @catehstn 

**Description**:
This PR injects JavaScript to override some common fingerprinting APIs and replaces their values with safer alternatives.

**Steps to test this PR**:
1. Visit https://privacy-test-pages.glitch.me/privacy-protections/fingerprinting/
1. Make sure the values returned match those listed in the Tech Design. (We are not overriding all values listed on the test page)
2. Visit https://duckduckgo.github.io/privacy-test-pages/
3. Verify no breakage on the test pages
4. Visit other well known websites (amazon.com, nytimes.com, youtube.com, etc.)
5. Verify app functionality works as expected



**Device Testing**:


**OS Testing**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
